bootstrap/000_setup_base_tools.sql

```diff
@@ -39,6 +39,13 @@ BEGIN
     return config_value;
 END;
 
+CREATE OR REPLACE PROCEDURE internal.has_config(key string) RETURNS BOOLEAN LANGUAGE SQL
+    AS
+BEGIN
+    let config_value string := (SELECT value FROM internal.config WHERE key = :key);
```

Comment:
can you return `internal.get_config(:key) is not null`? Just a bit less code dup

Reply:
I can definitely `call internal.get_config(:key)` to avoid the `select` query, but it doesn't seem like I can compress it down to a one-liner (have to call the proc into a var and then check the var)